terraform/environment/Makefile

```diff
@@ -39,3 +42,18 @@ else
 ENV_DIR=${CAILLEACH_DIR}/environments/${ENV}
 endif
 endif
+
+.PHONY: decrypt
+decrypt: $(ENV_DIR)/$(TOK).dec $(ENV_DIR)/$(SSH).dec
+
+$(ENV_DIR)/$(TOK).dec: check-env
```

Comment:
Nit: Looks like we're using round brackets and curly brackets in the same file. IIRC they mean the same things. Can we just be consistent?